src/components/event-info.tsx

```diff
@@ -9,21 +9,28 @@ export interface EventInfoProps {
   signUpLink: string;
 }
 
+const maxLength = 400
+
 export const EventInfo = (
   props: EventInfoProps,
 ): JSX.Element => {
+  const trimmedContent = props.content.length > maxLength ?
+    props.content.substring(0, maxLength - 3) + '... ' :
+    props.content;
+
```

Comment:
I believe you are trying to achieve the same as what a text-overflow CSS property would do? This is the [Tachyons solution](https://tachyons.io/docs/typography/measure/).

Reply:
I saw that tachyons solution but I think it's for horizontal overflow not vertical. Let me know if I'm missing something.